src/Book/Book.java

```diff
@@ -0,0 +1,99 @@
+package Book;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Book {
+    private final ISBN isbn;
+    private String title;
+    private Integer pages;
+    private Author author;
+    private BookType bookType;
+    private List<Review> reviews = new ArrayList<>();
+    private String publishingHouse;
+    private List<CopyOfBooks> copyOfBooks;
+
+    public void setPages(Integer pages) {
+        this.pages = pages;
+    }
+
+    public String getPublishingHouse() {
+        return publishingHouse;
+    }
+
+    public void setPublishingHouse(String publishingHouse) {
+        this.publishingHouse = publishingHouse;
+    }
+
+    public BookType getBookType() {
+        return bookType;
+    }
+
+    public void setBookType(BookType bookType) {
+        this.bookType = bookType;
+    }
+
+
+    public List<CopyOfBooks> getCopyOfBooks() {
+        return copyOfBooks;
+    }
+
+    public void setCopyOfBooks(List<CopyOfBooks> copyOfBooks) {
+        this.copyOfBooks = copyOfBooks;
+    }
+
+    public Book(ISBN isbn, String title, int pages, Author author, BookType bookType, String publishingHouse) {
```

Comment:
usually the constructory stays after the fields declaration, before setters and getters